container-multi-arch-manifest/README.md

```diff
@@ -38,7 +38,8 @@ jobs:
 | cosign-key-password | Cosign key password. Will be skipped if empty. Default is empty.                               | Required |
 | cosign-tlog-upload  | "Turn on or turn off the cosign tlog upload function. Options are true/false. Default is true. | Required |
 | digests             | New line seperated container image digest list.                                                | Required |
-| tags                | New line seperated multi-arch tag list.                                                        | Required |
+| tags                | New line seperated tag list. This or meta-tags are required. Default is empty.                 | Optional |
```

Comment:
One of both is definitely required so both being labeled as optional is only 50% of the truth.
But I quess it's fine cause there is no actual way to represent that condition.

Reply:
At least I didn't know about one.